cypress/integration/mmt-ui.spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+
+describe('mmt ui', () => {
+  beforeEach(() => {
+
+    const url = (() => {
+      const stage = Cypress.env('STAGE')
+      if (stage == "production")
+        return "https://mmt.maap-project.org"
+      else if (["main", "dit"].includes(stage) || stage.startsWith("refs"))
```

Comment:
I'm curious where the stage identifier of `main` comes from. Are there separate stages for `production` and `main`?

I also don't understand what this means: `stage.startsWith("refs")`

Reply:
refs is a git convention, IE "refs/head/<branch>". As this set of tests are triggered by the deployment of other repos.

Reply:
ref: https://aboullaite.me/deep-dive-into-git-git-refs/#:~:text=Git%20also%20has%20a%20special,to%20refs%2Fhead%2Fmaster%20.&text=It%20is%20actually%20possible%20for,directly%20to%20a%20commit%20object.

Reply:
RE: Main/stages